src/nbb/core.cljs

```diff
@@ -159,7 +169,12 @@
           (load-module "./nbb_tools_cli.js" libname as refer rename libspecs)
           (goog.string goog.string.format)
           (load-module "./nbb_goog_string.js" libname as refer rename libspecs)
-          (if (string? libname)
+          (cognitect.transit)
+          (load-module "./nbb_transit.js" libname as refer rename libspecs)
+          (cond
+            (get (feature-requires) libname)
+            (load-module (get (feature-requires) libname) libname as refer rename libspecs)
```

Comment:
We could generalize the usage of this requires map to non-features namespaces like the above e.g. `goog.string "./nbb_goog_string.js"`

Reply:
We can do that later if we find that the current approach works to our satisfaction.